h/static/scripts/hooks/use-group-annotations.tsx

```diff
@@ -85,6 +91,12 @@ export function useGroupAnnotations({
     new Set<string>(),
   );
 
+  // We are assuming `removedAnnotations` will only include annotations that
+  // have been previously loaded and therefore are part of `annotations`.
+  const visibleAnnotations = annotations
+    ? annotations.length - removedAnnotations.size
+    : 0;
```

Comment:
We could do this directly in `GroupModeration`, but this is going to be used here in https://github.com/hypothesis/h/pull/9838